From f50b88d4be286d744361f7dadff0e6ae8ba107ff Mon Sep 17 00:00:00 2001
From: Anand Balagopalakrishnan <anandb@ti.com>
Date: Sat, 15 Aug 2015 22:37:39 +0000
Subject: [PATCH 2/5] ARM: dts: DRA7xx: add gpu interface clock

Addition of SGX interface sys clock to DRA7xx device tree.

Signed-off-by: Anand Balagopalakrishnan <anandb@ti.com>
---
 arch/arm/boot/dts/dra7xx-clocks.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/dra7xx-clocks.dtsi b/arch/arm/boot/dts/dra7xx-clocks.dtsi
index bd11c46..c734f75 100644
--- a/arch/arm/boot/dts/dra7xx-clocks.dtsi
+++ b/arch/arm/boot/dts/dra7xx-clocks.dtsi
@@ -753,6 +753,14 @@
 		ti,index-power-of-two;
 	};
 
+	gpu_l3_iclk: gpu_l3_iclk {
+		#clock-cells = <0>;
+		compatible = "fixed-factor-clock";
+		clocks = <&l3_iclk_div>;
+		clock-mult = <1>;
+		clock-div = <1>;
+	};
+
 	l4_root_clk_div: l4_root_clk_div {
 		#clock-cells = <0>;
 		compatible = "fixed-factor-clock";
-- 
2.2.0

